### DISCUSSION TITLE
How to pass the custom values yaml file for helm remote chart install?

### DISCUSSION THREAD

**Discussion by rameshar16** [emacs-helm/helm#2513] [General]
I am trying to install the helm chart from the remote repo and trying to override the default values.yaml file with custom values yaml file but the helm is unable to find the custom values yaml file.

$ helm install -n dev app repo/app-f values.dev.yaml
Error: open values.dev.yaml: no such file or directory

» helm version
version.BuildInfo{Version:"v3.8.1", GitCommit:"5cb9af4b1b271d11d7a97a71df3ac337dd94ad37", GitTreeState:"clean", GoVersion:"go1.17.8"}

**Comment by thierryvolpiatto** [comment]
rameshar16 ***@***.***> writes:

> I am trying to install the helm chart from the remote repo and trying to override the default values.yaml file with custom values yaml file but the helm is unable to find the custom values
> yaml file.

Are you sure you are speaking of Helm, the emacs-helm extension for
emacs?


-- 
Thierry

**Reply by rameshar16** [reply]
I am asking about the helm? not about the emacs-helm.

**Comment by thierryvolpiatto** [comment]
rameshar16 ***@***.***> writes:

> I am asking about the helm? not about the emacs-helm.

What helm ? You are here on emacs-helm github discussion.

> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.*Message ID: ***@***.***>


-- 
Thierry
